zyte_spider_templates/spiders/ecommerce.py

```diff
@@ -30,13 +31,17 @@ class EcommerceSpiderParams(BaseSpiderParams):
         default=EcommerceCrawlStrategy.navigation,
         json_schema_extra={
             "enumMeta": {
+                EcommerceCrawlStrategy.full: {
+                    "title": "Full",
+                    "description": "Follow most links within the domain of URL in an attempt to discover and extract as many products as possible.",
+                },
                 EcommerceCrawlStrategy.navigation: {
                     "title": "Navigation",
                     "description": "Follow pagination, subcategories, and product detail pages.",
                 },
-                EcommerceCrawlStrategy.full: {
-                    "title": "Full",
-                    "description": "Follow most links within the domain of URL in an attempt to discover and extract as many products as possible.",
+                EcommerceCrawlStrategy.pagination_only: {
+                    "title": "Pagination Only",
+                    "description": "Follow pagination and product detail pages.",
```

Comment:
What do you think about adding some motivation to the description here, similar to the PR description?

Reply:
Good idea. Updated in https://github.com/zytedata/zyte-spider-templates/pull/2/commits/396283d82f2a92175698d3cbd4f0dd1012310af5